index.html

```diff
@@ -181,8 +182,9 @@ <h2>Scope</h2>
         <p>The Working Group may also publish new Working Group Notes that may be necessary to clarify, help the deployment and usage, or propose new features to the Group’s main recommendation-track deliverable.</p>
         <p>
 The Working Group may also publish new Working Group Notes, Editors Drafts, or
-Working Drafts for DID Resolution or of DID Method Specifications. Any documents
-intended for Recommendation will be limited to the Working Draft maturity level.
+Working Drafts for DID Resolution, DID URL Dereferencing, or of DID Method
```

Comment:
This change could be expanded to as `Working Drafts for DID Resolution or for DID URL Dereferencing, or of DID Method`
```suggestion
Working Drafts for DID Resolution or DID URL Dereferencing, or of DID Method
```